adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -473,6 +507,25 @@ def connect_AP(self, ssid, password): # pylint: disable=invalid-name
             raise RuntimeError("No such ssid", ssid)
         raise RuntimeError("Unknown error 0x%02X" % stat)
 
+    def create_ap(self, ssid, password, channel=b'\x01'):
+        """Create an access point with the given name and password."""
+        if isinstance(ssid, str):
+            ssid = bytes(ssid, 'utf-8')
+        if password:
+            if isinstance(password, str):
+                password = bytes(password, 'utf-8')
+            self.wifi_set_ap_passphrase(ssid, password, channel)
+        else:
+            self.wifi_set_ap_network(ssid, channel)
+        for _ in range(10): # retries
```

Comment:
You might want to implement your retry loop as time-based rather than retry-based. You may also want to consider making the number of retries (10) a variable kwarg named `retries`.

Reply:
Great suggestion, thanks. Done.